gcov2lcov.go

```diff
@@ -40,24 +40,31 @@ type cacheEntry struct {
 
 var pkgCache = map[string]cacheEntry{}
 
-func ConvertCoverage(in io.Reader, out io.Writer, pathResolverFunc func(string) string) error {
+type PathResolver func(name string) (string, error)
+
+func ConvertCoverage(in io.Reader, out io.Writer, pathResolverFunc PathResolver) error {
 	blocks, err := parseCoverage(in, pathResolverFunc)
 	if err != nil {
 		return err
 	}
 	return writeLcov(blocks, out)
 }
 
-func AbsolutePathResolver(name string) string {
-	return name
+func AbsolutePathResolver(name string) (string, error) {
```

Comment:
I think the name is a little bit misleading and should have been named something like `IdentityPathResolver` since the function does not actually change or make a path absolute (as can be seen in the test)